### PR TITLE
Refactor to new version of the core

### PR DESCRIPTION
**Pull request by MahdiyarGHD**
### Checklist:

- [ ] It refers to an [Issue](https://github.com/EasyMicroservices/FileManager/issues).
- [ ] It fixes a bug.
- [ ] It's a new feature (non-breaking change which adds functionality)
- [ ] It's a breaking (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.
- [ ] Related language label is selected.

### CODE REVIEW

File: src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormController.cs [hunk 1]
```diff
@@ -11,10 +12,10 @@ namespace EasyMicroservices.TemplateGeneratorMicroservice.WebApi.Controllers
     public class FormController : SimpleQueryServiceController<FormEntity, CreateFormRequestContract, FormContract, FormContract, long>
     {
         IContractLogic<FormItemEntity, FormItemEntity, FormItemEntity, FormItemEntity, long> _formItemContractReadable;
-        public FormController(IContractLogic<FormEntity, CreateFormRequestContract, FormContract, FormContract, long> contractReadable,
-            IContractLogic<FormItemEntity, FormItemEntity, FormItemEntity, FormItemEntity, long> formItemContractReadable) : base(contractReadable)
+
+        public FormController(IUnitOfWork uow) : base(uow)
         {
-            _formItemContractReadable = formItemContractReadable;
+            _formItemContractReadable = uow.GetReadableContractLogic<FormItemEntity, FormItemEntity, long>();
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormController.cs]: this code will not let you write (add, update and delete)

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

File: src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormValuesController.cs [hunk 2]
```diff
@@ -13,10 +14,11 @@ public class FormValuesController : SimpleQueryServiceController<FormFilledEntit
     {
         IEasyReadableQueryableAsync<FormItemValueEntity> _formValueContractReadable;
         IEasyReadableQueryableAsync<FormItemEntity> _formItemContractReadable;
-        public FormValuesController(IContractLogic<FormFilledEntity, FormValuesContract, FormValuesContract, FormValuesContract, long> contractReadable, IEasyReadableQueryableAsync<FormItemValueEntity> formValueContractReadable, IEasyReadableQueryableAsync<FormItemEntity> formItemContractReadable) : base(contractReadable)
+
+        public FormValuesController(IUnitOfWork uow) : base(uow)
         {
-            _formValueContractReadable = formValueContractReadable;
-            _formItemContractReadable = formItemContractReadable;
+            _formValueContractReadable = uow.GetReadableOf<FormItemValueEntity>();
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormValuesController.cs:20]: this code will not let you write (add, update, and delete)

File: src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormValuesController.cs [hunk 3]
```diff
@@ -13,10 +14,11 @@ public class FormValuesController : SimpleQueryServiceController<FormFilledEntit
...
         IEasyReadableQueryableAsync<FormItemValueEntity> _formValueContractReadable;
         IEasyReadableQueryableAsync<FormItemEntity> _formItemContractReadable;
-        public FormValuesController(IContractLogic<FormFilledEntity, FormValuesContract, FormValuesContract, FormValuesContract, long> contractReadable, IEasyReadableQueryableAsync<FormItemValueEntity> formValueContractReadable, IEasyReadableQueryableAsync<FormItemEntity> formItemContractReadable) : base(contractReadable)
+
+        public FormValuesController(IUnitOfWork uow) : base(uow)
         {
-            _formValueContractReadable = formValueContractReadable;
-            _formItemContractReadable = formItemContractReadable;
+            _formValueContractReadable = uow.GetReadableOf<FormItemValueEntity>(); 
+            _formItemContractReadable = uow.GetReadableOf<FormItemEntity>(); ;
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.TemplateGeneratorMicroservice.WebApi/Controllers/FormValuesController.cs:21]: this code will not let you write (add, update, and delete)

**Review by Elyas-Amini — APPROVED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]

**Review by mahdidelzendeh72 — COMMENTED** [review]
thanks

**Pull request merged**